tests/desktop/consumer_pages/test_home_page.py

```diff
@@ -34,7 +34,7 @@ def test_that_header_has_expected_items(self, mozwebqa):
 
         Assert.true(home_page.header.is_logo_visible)
         Assert.true(home_page.header.is_search_visible)
-        Assert.equal(home_page.header.search_field_placeholder, "Search the Marketplace")
+        Assert.equal(home_page.header.search_field_placeholder, u'Search Marketplace\u2026')
```

Comment:
Nit, can you add a comment pertaining to the importance of this unicode character?